### PR TITLE
Create 2015-10-13-fukuoka-ruby-award-2016.md

### PR DESCRIPTION
**Pull request by digitalfukuoka**

### CODE REVIEW

File: en/news/_posts/2015-10-13-fukuoka-ruby-award-2015.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+The Government of Fukuoka, Japan together with "Matz" Matsumoto would
+like to invite you to enter the following Ruby competition. If you have developed an interesting Ruby program, please be encouraged to apply.
++
+2015 Fukuoka Ruby Award Competition
+ - Grand Prize - 1 Million Yen!
+
+Entry Deadline: November 27, 2015
+
+Matz and a group of panelists will select the winners of the Fukuoka Competition.
```
Comment (yous) [inline en/news/_posts/2015-10-13-fukuoka-ruby-award-2015.md]: There is a trailing whitespace.

File: en/news/_posts/2015-10-13-fukuoka-ruby-award-2015.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+lang: en
+---
+
+Dear Ruby Enthusiasts,
+
+
+The Government of Fukuoka, Japan together with "Matz" Matsumoto would
+like to invite you to enter the following Ruby competition. If you have developed an interesting Ruby program, please be encouraged to apply.
++
+2015 Fukuoka Ruby Award Competition
```
Comment (yous) [inline en/news/_posts/2015-10-13-fukuoka-ruby-award-2015.md]: Should this year be 2016?

**Comment by yous** [comment]
Should the filename be `2015-10-13-fukuoka-ruby-award-2016.md`?

File: en/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+title: "2016 Fukuoka Ruby Award Competition -Entries to be judged by Matz"
+author: "Fukuoka Ruby"
+translator:
+date: 2015-10-13 17:00:00 +0900
+lang: en
+---
+
+Dear Ruby Enthusiasts,
+
+
```
Comment (JuanitoFatas) [inline en/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md]: Extra blank line detected.

File: en/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+The winner of the GMO Pepabo Prize will receive:
+- 50,000 yen gift certificate toward domain services from MuuMuu Domain
+- Gift basket filled with local foods and snacks (30,000 yen value)
+
+The winner of the IIJ GIO Prize will receive:
+- IIJ GIO free coupon worth 500,000 yen(Up to 6 months)
+
+"Matz will be testing and reviewing your source code thoroughly, so it's very meaningful to apply! The competition is free to enter."
+
+
```
Comment (JuanitoFatas) [inline en/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md]: Extra blank line detected.

File: en/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+Entry Deadline: November 27, 2015
+
+Matz and a group of panelists will select the winners of the Fukuoka Competition. 
+The grand prize for the Fukuoka Competition is 1 million yen. Past grand prize winners include Rhomobile (USA) and APEC Climate Center (Korea).
+
+http://myfukuoka.com/category/news/ruby-news/
+
+Programs entered in the competition do not have to be written entirely in Ruby
+ but should take advantage of the unique characteristics of Ruby.
```
Comment (JuanitoFatas) [inline en/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md]: Remove space before `but`?

File: en/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+http://www.digitalfukuoka.jp/events/84
+or
+http://myfukuoka.com/events/2016-fukuoka-ruby-award-guidelines-for-applicants/
+
+http://www.digitalfukuoka.jp/uploads/event_detail/file/210/RubyAward_ApplicationForm_2016.doc
+
+Please email the application form to award@f-ruby.com
+
+This year, we have the following special prizes:
+The winner of the Engine Yard Prize will receive:
```
Comment (JuanitoFatas) [inline en/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md]: Should these two lines be: ```This year, we have the following special prizes. The winner of the Engine Yard Prize will receive:```?

File: en/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+Projects must have been developed or completed within the past 12 months to be eligible. Please visit the following Fukuoka website for additional details or to enter:
+
+http://www.digitalfukuoka.jp/events/84
+or
+http://myfukuoka.com/events/2016-fukuoka-ruby-award-guidelines-for-applicants/
+
+http://www.digitalfukuoka.jp/uploads/event_detail/file/210/RubyAward_ApplicationForm_2016.doc
+
+Please email the application form to award@f-ruby.com
```
Comment (JuanitoFatas) [inline en/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md]: Missing a period at the end of sentence.

**Comment by JuanitoFatas** [comment]
@zzak Could you take a look? :bow:

**Comment by zzak** [comment]
@JuanitoFatas I think these changes are small enough that we can make them after we merge the PR

**Comment by JuanitoFatas** [comment]
> @JuanitoFatas I think these changes are small enough that we can make them after we merge the PR

:ok_hand: Thanks for your reply :heart:

**Comment by yous** [comment]
LGTM

File: en/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
+---
+layout: news_post
+title: "2016 Fukuoka Ruby Award Competition -Entries to be judged by Matz"
```
Comment (hlcfan) [inline en/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md]: It looks better if there's a space before `Entries` 
"2016 Fukuoka Ruby Award Competition - Entries to be judged by Matz"

**Comment by hlcfan** [comment]
Looks good to me except that minor snag.

**Comment by hlcfan** [comment]
:shipit:

File: en/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+The winner of the GMO Pepabo Prize will receive:
+- 50,000 yen gift certificate toward domain services from MuuMuu Domain
+- Gift basket filled with local foods and snacks (30,000 yen value)
+
+The winner of the IIJ GIO Prize will receive:
+- IIJ GIO free coupon worth 500,000 yen(Up to 6 months)
+
+"Matz will be testing and reviewing your source code thoroughly, so it's very meaningful to apply! The competition is free to enter."
+
+Cheers,
```
Comment (stomar) [inline en/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md]: `Cheers,`

from whom?

**Pull request merged**